### PR TITLE
Add views to blog app

### PR DESCRIPTION
**Pull request by martinkarugaba**
# :recycle:  Changes made
## Add views for;
- [ ] List of users
- [ ] User details
- [ ] User posts
- [ ] User post details

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,4 +1,44 @@
...
+    </div>
+
+    <div class='border-2 border-black mt-8'>
+      <p>Bio: <%= @user.bio %></p>
+    </div>
+
+    <div class='mt-8 flex flex-col justify-start start-center gap-4'>
+      <h2 class='font-medium'>Posts by <%= @user.name %></h2>
+      <% @user.posts.limit(3).each do |post| %>
+        <div class="post border border-black">
```
Comment (ShadyShawkat) [inline app/views/users/show.html.erb]: - [ ] Please make sure to use methods that you have created in the models, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.) requirement.

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1,12 +1,53 @@
...
+        <div class="post border border-black">
+          <h3><%= post.title %></h3>
+          <p><%= post.text %></p>
+          <div class='mt-4 flex justify-end items-center gap-4 text-sm'>
+            <p>Number of Comments: <%= post.comments.count %></p>
+            <p>Number of Likes: <%= post.likes.count %></p>
+          </div>
+          <ul class='border-2'>
+            <% post.comments.order(created_at: :desc).limit(5).each do |comment| %>
+              <li><%= comment.text %></li>
```
Comment (ShadyShawkat) [inline app/views/posts/index.html.erb]: - [ ] Please make sure to use methods that you have created in the models, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.) requirement.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @martinkarugaba, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @martinkarugaba , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Good work utilizing your `Model` private methods  👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**